src/components/picker/index.js

```diff
@@ -385,10 +386,12 @@ class Picker extends PureComponent {
 
     const textInputProps = TextField.extractOwnProps(this.props);
     const label = this.getLabelValueText();
-    
+    const layoutProps = {...extractMarginValues(this.props), ...extractPaddingValues(this.props)};
```

Comment:
no need for that.. 
by wrapping a component with asBaseComponent you get injected with the `modifiers` prop. 
From there you can get whatever modifiers values you need (margins, paddings, colors, etc..) 

I would see how `View` is implemented for a reference.

Reply:
Done.